modules/iasRtdProvider.js

```diff
@@ -31,11 +36,24 @@ function stringifySlotSizes(sizes) {
   return result;
 }
 
-function stringifySlot(bidRequest, adUnitPath) {
+function getAllSlots() {
+  return utils.isGptPubadsDefined() && window.googletag.pubads().getSlots();
+}
+
+function getSlotByCode(code) {
+  const slots = getAllSlots();
```

Comment:
The `getAllSlots` function is repeating work done by the `utils.getGptSlotInfoForAdUnitCode` function. You could likely remove the former and instead check for an empty object returned from the latter.